scripts/docker/centos.7/Dockerfile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM centos:7.1.1503
+
+RUN yum -q -y install deltarpm
```

Comment:
The commit message for the commit that introduced this made this clear, but GitHub makes it hard to see such a thing.  I wanted to have the directories named after their rid to simplify logic in dockerbuild.sh, but since the VSO jobs were using the old names, I had to copy these around in the interm.